pythonscript/_godot_script.pxi

```diff
@@ -194,4 +219,10 @@ cdef api godot_pluginscript_script_manifest pythonscript_script_init(
 cdef api void pythonscript_script_finish(
     godot_pluginscript_script_data *p_data
 ) with gil:
-    destroy_exposed_class(<object>p_data)
+    cdef object cls = <object>p_data
+    destroy_module(cls=cls)
+
+cdef inline destroy_module(cls=None):
+    if get_pythonscript_verbose():
+        print(f"Destroying python script")
```

Comment:
Not sure it's the right place to put this print: when we call this function from the reloading code we end up with first a print about reloading the module, then this print about deleting a script...
I think it would be better to have the print about deleting the script only in `pythonscript_script_finish` and juste have a loading or reloading print on the `pythonscript_script_init`

And this delete print should contain the name of the module ;-)